indent/ruby.vim

```diff
@@ -182,7 +187,10 @@ function GetRubyIndent(...)
     let indent = call(function(callback_name), [indent_info])
 
     if indent >= 0
-"      Decho "Match: ".callback_name
+"      Decho "Match: ".callback_name." indent=".indent
```

Comment:
The reason I put the debug statement is just to know which of the callbacks matched. I don't care that much about the indent value, because, when debugging, I call `:echo GetRubyIndent(<linenumber>)`. This outputs the indent value, and dumps debug info about what happened.

I'm still okay with adding the indent there, but the rest of the comments are a bit too much. If nothing else, they wouldn't really work. The `plnum` is added below, so, these debug statements, if hit, will generate errors. I'm assuming you tested it with one of the below callbacks matching, which is why it worked, but if you enable debugging, and check, say, a closing bracket on an empty line (just a `}` on its own), you'll get a lot of errors about undefined keys.

I do appreciate the thought, and it does make sense to get some more info. What I'd suggest is that you replace it with something like this:

``` vim
Decho "Match: ".callback_name." info=".string(indent_info)
```

This would be pretty helpful, I think, and it would be much less code to get the same amount of information.